### PR TITLE
mimix1050rt: gPTP build fix

### PR DESCRIPTION
**Pull request by agansari**
Sets iMX RT's cmake and fixup to enable gPTP in MCUX_ETH.

Signed-off-by: Andrei Gansari <andrei.gansari@nxp.com>

Fixes an issue observed by @jeremy-e-mills in #11586

### CODE REVIEW

**Review by dbkinder — APPROVED** [review]
+1 for doc change

**Comment by agansari** [comment]
@MaureenHelm please have a look at this?

File: samples/net/gptp/README.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ Follow these steps to build the gPTP sample application:
 .. zephyr-app-commands::
    :zephyr-app: samples/net/gptp
    :board: <board to use>
-   :conf: prj.conf
+   :conf: prj_base.conf
```
Comment (MaureenHelm) [inline samples/net/gptp/README.rst]: Why are you changing this?

Comment (jukkar) [reply samples/net/gptp/README.rst]: Good catch, but that should be placed in separate commit as it is unrelated to mimix1050rt support.

**Review by jukkar — CHANGES_REQUESTED** [review]

**Review by agansari — COMMENTED** [review]
@jukkar @MaureenHelm split documentation update to a separate commit
I added another minor documentation fix to close #11531

**Review by jukkar — APPROVED** [review]
LGTM

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**